sources/packages/backend/libs/services/src/application/application.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from "@nestjs/common";
+import { Repository } from "typeorm";
+import { Application } from "@sims/sims-db";
+import { InjectRepository } from "@nestjs/typeorm";
+
+/**
+ * Service layer for Application.
+ */
+@Injectable()
+export class ApplicationService {
+  constructor(
+    @InjectRepository(Application)
+    private readonly applicationRepo: Repository<Application>,
+  ) {}
+
+  /**
+   * Checks if there was multiple submission for an application
+   * (i.e student edits an application after submission).
+   * @param applicationNumber application number.
+   * @returns true if there was multiple submission for
+   * an application.
+   */
+  async hasMultipleApplicationSubmissions(
+    applicationNumber: string,
+  ): Promise<boolean> {
+    const applicationSubmissions = await this.applicationRepo.find({
```

Comment:
we can use the query to directly return the result instead of a select. 

```sql
select 1 from sims.applications a where a.application_number = '2023000002' group by a.application_number having count(a.application_number) > 1 ; 
```

we can do this query builder 

or use the count method. 

```ts
const applicationSubmissions = await this.applicationRepo.count({
      select: { id: true },
      where: {
        applicationNumber,
      },
    });
```